README.md

```diff
@@ -253,44 +246,78 @@ If your desired ghc has been found, you use it to install haskell-language-serve
 ./cabal-hls-install data
 ```
 
-## HLS LSP Configuration
+## Configuring `haskell-language-server`
 
-haskell-language-server supports some forms of configuration.
+Language servers like `haskell-language-server` expose most of their configuration via the client (i.e. the editor).
+That means that the way in which you configure the settings will depend on the client.
 
-This configuration is done via the LSP settings you provide with your editor and/or LSP client.
-Some examples include:
-* in VSCode/VSCodium this is done via the `Settings` tab
-* with `LanguageClient-neovim` you can point the [`g:LanguageClient_settingsPath`](https://github.com/autozimu/LanguageClient-neovim/blob/0e5c9546bfddbaa2b01e5056389c25aefc8bf989/doc/LanguageClient.txt#L221)
-  variable to the file in which you want to keep your LSP settings
+Most clients (editors) already have an opinion about how settings should be configured!
+For example, in VS Code you use the graphical Settings tab or `settings.json`, whereas in Emacs you use customization variables.
+In the [editor configuration section](#configuring-your-editor) we give some pointers for popular editors, but you should consult the documentation for your specific editor if you have trouble.
```

Comment:
The "pointers" are missing, I think, so the first part of the sentence is a bit confusing.

Reply:
Hm yes, it's not very explicit, let's fix that.

Reply:
Ah, sorry! My original comment was kind of dumb in retrospect, I hadn't noticed the markdown formatting!
In any case, thanks for adding more stuff, I would "resolve" this if I was more adequate at github!